source/jormungandr/jormungandr/authentication.py

```diff
@@ -135,26 +135,6 @@ def has_access(region, api, abort, user):
         else:
             return False
 
-@cache.memoize(current_app.config['CACHE_CONFIGURATION'].get('TIMEOUT_AUTHENTICATION', 300))
-def has_access_to_global_places(user):
-    """
-    Since there is only open data in the global /places by default, to access the global places the user
-    should be either a super user or have access to the open data
-    """
-    if current_app.config.get('PUBLIC', False):
-        #if jormungandr is on public mode we skip the authentification process
-        return True
-    if not user:
-        # a user is mandatory even for free region
-        return False
-    return user.is_super_user or user.have_access_to_free_instances
```

Comment:
Now sandbow token is accessing places... We don't want that, and we have to discuss a bit about that (probably just authorize if you have access to open data).
If we let it that way, we may have to clean corresponding part of models.py in navitiacommon.